modules/standard/online/log_controller.py

```diff
@@ -1,56 +1,75 @@
-from core import db, command_request
-from core.command_param_types import Any
+from core.command_param_types import Any, Const
 from core.command_request import CommandRequest
 from core.db import DB
-from core.decorators import instance, command, event
+from core.decorators import instance, command
+
 
 @instance()
 class LogController:
 
     def inject(self, registry):
-        self.bot = registry.get_instance("bot")
         self.db: DB = registry.get_instance("db")
-        self.text = registry.get_instance("text")
-        self.util = registry.get_instance("util")
-        self.pork_service = registry.get_instance("pork_service")
-        self.character_service = registry.get_instance("character_service")
-        self.discord_controller = registry.get_instance("discord_controller")
-        self.command_alias_service = registry.get_instance("command_alias_service")
-        self.alts_service = registry.get_instance("alts_service")
-        self.alts_controller = registry.get_instance("alts_controller")
-
-    @command(command="logon", params=[Any("logon_message")], access_level="member",
-             description="Sets your own custom logon message")
-    def set_logon(self, request: CommandRequest, logon_message):
+
+    @command(command="logon", params=[Const("clear")], access_level="member", description="Clear your logon message.")
+    def clear_logon(self, request, params):
         if self.db.query_single("SELECT logon FROM log_messages WHERE char_id=?;", [request.sender.char_id]):
-            self.db.query_single("UPDATE log_messages SET logon=? WHERE char_id=?;",
-                                 [logon_message, request.sender.char_id])
-        else:
-            self.db.query_single("INSERT INTO log_messages (char_id, logon) VALUES(?, ?);",
-                                 [request.sender.char_id,  logon_message])
-        return "Your new logon message is: %s" % logon_message
+            self.db.query_single("UPDATE log_messages SET logon=NULL WHERE char_id=?;",
```

Comment:
so everywhere that there is an UPDATE or INSERT or DELETE we should use `self.db.exec()` instead of `query_single`.  I missed that on the PR that added this code